src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,3 @@
-// 
+.card-image img {
```

Comment:
It's better to avoid using tag selectors. You can create a separate class name for an image.